### ISSUE TITLE
request: inform user about which species had a successfull estimation of parameters

### ISSUE THREAD

**Issue by CeresBarros** [ianmseddy/LandR_speciesParameters#9]
Could we add a message in PSP to informing the user about which species have had parameters estimated from PSP?
Right now we have to "manually" compare the `species` table with the original LANDIS-II one which is not very straightforward, especially for newbies.

**Comment by CeresBarros** [comment]
As of now, species that have enough data to fit GAMMS will get parameters estimated from PSP (regardless of the quality of the said GAMMs). Hence the following message is being printed at present:
```
2021-02-19 01:30:59 INFO::building growth curves from PSP data for these species:
[1] "Betu_Pap" "Lari_Lar" "Pice_Gla" "Pice_Mar" "Pinu_Ban" "Popu_Spp"
```
this means that the listed species will have traits being estimated.
If any species does not have enough data to estimate traits, a warning is issued saying GAMMs could not be fit.

I believe a simple tweak of the message will suffice to make this all clearer - will PR soon :)

**Comment by CeresBarros** [comment]
fixed in f2486711b8b4ae644619a7d1aa5a0bcf3e2e3ddd
